cmd/create.go

```diff
@@ -0,0 +1,245 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	log "github.com/charmbracelet/log"
+	"github.com/spf13/cobra"
+	"github.com/thi-startup/spitfire/internal/config"
+	"github.com/thi-startup/spitfire/internal/drive"
+	"github.com/thi-startup/spitfire/internal/get"
+	"github.com/thi-startup/spitfire/utils"
+	"golang.org/x/net/context"
+)
+
+type rootfs struct {
+	size   string
+	fstype string
+	name   string
+	image  string
+	init   bool
+	path   string
+}
+
+func MakeCreateCmd() *cobra.Command {
+	command := &cobra.Command{
+		Use:          "create [flags] <image>",
+		Short:        "create a microvm",
```

Comment:
- is the create's not too specific? what if you want to create only the loop drive??
- what the command mainly does is create the loop drive, with options to add an image, init or kernel or whatever we might want to add in there in the future

- i think "create vm drive" might be a better description for the command.